src/main/java/mate/academy/dao/OrderDao.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.dao;
+
+import java.util.List;
+import mate.academy.model.Order;
+import mate.academy.model.User;
+
+public interface OrderDao {
+
+    Order add(Order order);
+
+    Order getByUser(User user);

```

Comment:
Method Naming Convention: The method `getByUser` should be renamed to `getOrdersByUser` to follow Java's camelCase naming convention and to clearly indicate its purpose of retrieving orders by user.